### DISCUSSION TITLE
Github account gets disabled when I integrate it with CircleCI

### DISCUSSION THREAD

**Discussion by sunitanair30** [community/community#59848] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Every time I try to connect my account to CircleCI, my github account gets disabled. 
I use the authorize circleCI option from within CircleCI.
Is there a different option to allow CircleCI to integrate with my free github account?

**Comment by hrishikesh-kocharekar-dev** [comment]
Verify CircleCI settings: Double-check the settings within CircleCI to ensure that you are following the correct steps to authorize the integration. CircleCI provides documentation and guides on how to connect your GitHub account, so make sure you are following their instructions correctly.

**Reply by sunitanair30** [reply]
I did. There are no settings to configure the connectivity to github
